mobile-browser-based-version/src/helpers/training_script/custom_layers.js

```diff
@@ -0,0 +1,81 @@
+import * as tf from '@tensorflow/tfjs';
+
+/*
+ * This calss defines a new type of Layer for Interoperability personalization
```

Comment:
class. and give a bit more details so it's clear, cite the paper also.
is it only meant for the input layer (i guess so) or also personal output layer?

Reply:
It's meant for both personal input and output layers.